generate_endf71.py

```diff
@@ -99,122 +100,200 @@ def sort_key(path):
-]
-wmp_version = '1.1'
-wmp_base = f'https://github.com/mit-crpg/WMP_Library/releases/download/v{wmp_version}/'
-wmp_filename = f'WMP_Library_v{wmp_version}.tar.gz'
-
-
-neutron_dir = Path('neutrons')
-thermal_dir = Path('thermal_scatt')
-thermal_paths = [
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinH2O.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinCH2.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinZrH.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-ortho-H.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-para-H.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-benzine.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-l-CH4.endf'),
-    (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-s-CH4.endf'),
-    (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-DinD2O.endf'),
-    (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-ortho-D.endf'),
-    (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-para-D.endf'),
-    (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-BeinBeO.endf'),
-    (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-Be-metal.endf'),
-    (neutron_dir / 'n-006_C_000.endf', thermal_dir / 'tsl-graphite.endf'),
-    (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinBeO.endf'),
-    (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinUO2.endf'),
-    (neutron_dir / 'n-013_Al_027.endf', thermal_dir / 'tsl-013_Al_027.endf'),
-    (neutron_dir / 'n-026_Fe_056.endf', thermal_dir / 'tsl-026_Fe_056.endf'),
-    (neutron_dir / 'n-014_Si_028.endf', thermal_dir / 'tsl-SiO2.endf'),
-    (neutron_dir / 'n-040_Zr_090.endf', thermal_dir / 'tsl-ZrinZrH.endf'),
-    (neutron_dir / 'n-092_U_238.endf', thermal_dir / 'tsl-UinUO2.endf')
-]
-
-pwd = Path.cwd()
-
-(args.destination / 'photon').mkdir(parents=True, exist_ok=True)
-(args.destination / 'wmp').mkdir(parents=True, exist_ok=True)
-
-with tempfile.TemporaryDirectory() as tmpdir:
-    # Save current working directory and temporarily change dir
-    if args.tmpdir:
-        os.chdir(tmpdir)
-    library = openmc.data.DataLibrary()
-
-    # =========================================================================
-    # Download files from NNDC server
-    if args.download:
-        for base, fname, checksum in files:
-            download(urljoin(base, fname), checksum)
-
-    # =========================================================================
-    # EXTRACT FROM ZIP FILES
-
-    if args.extract:
-        for _, f, _ in files:
-            print(f'Extracting {f}...')
-            zipfile.ZipFile(f).extractall()
-
-    # =========================================================================
-    # PROCESS INCIDENT NEUTRON AND THERMAL SCATTERING DATA IN PARALLEL
+library_name = 'endf'
+release = 'b7.1'
+
+cwd = Path.cwd()
+
+wmp_files_dir = args.destination / 'neutron_wmp'
```

Comment:
To stay consistent with current behavior
```suggestion
wmp_files_dir = args.destination / 'wmp'
```